internal/gatewayapi/backendtrafficpolicy.go

```diff
@@ -0,0 +1,338 @@
+
+				status.SetBackendTrafficPolicyCondition(policy,
+					egv1a1.PolicyConditionOverridden,
+					metav1.ConditionTrue,
+					egv1a1.PolicyReasonOverridden,
+					message,
+				)
+			}
+
+			// Add section to targeted gateways map
+			if s == nil {
+				targetedGateways[*gatewayKey] = sets.New[string]()
+			}
+			targetedGateways[*gatewayKey].Insert(section)
+
+			translateBackendTrafficPolicy(policy, xdsIR)
+
+			// Set Accepted=True
+			status.SetBackendTrafficPolicyCondition(policy,
+				gwv1a2.PolicyConditionAccepted,
+				metav1.ConditionTrue,
+				gwv1a2.PolicyReasonAccepted,
+				"BackendTrafficPolicy has been accepted.",
+			)
+		}
+	}
+
+	// Process the policies targeting Gateways with no section name set
+	for _, policy := range backendTrafficPolicies {
+		if policy.Spec.TargetRef.SectionName == nil && policy.Spec.TargetRef.Kind == KindGateway {
+
+			policy := policy.DeepCopy()
+			res = append(res, policy)
+
+			// Negative statuses have already been assigned so its safe to skip
+			gatewayKey := resolveBTPolicyTargetRef(policy, gatewayMap, routeMap)
+			if gatewayKey == nil {
+				continue
+			}
+
+			// Check if another policy targeting the same Gateway exists
+			s, ok := targetedGateways[*gatewayKey]
+			if ok && s.Has(AllSections) {
+				message := "Unable to target Gateway, another BackendTrafficPolicy has already attached to it"
+
+				status.SetBackendTrafficPolicyCondition(policy,
+					gwv1a2.PolicyConditionAccepted,
+					metav1.ConditionFalse,
+					gwv1a2.PolicyReasonConflicted,
+					message,
+				)
+
+				continue
+
+			}
+
+			if ok && (s.Len() > 0) {
+				// Maintain order here to ensure status/string does not change with same data
+				sections := s.UnsortedList()
+				sort.Strings(sections)
+				message := fmt.Sprintf("There are existing BackendTrafficPolicies that are overriding these sections %v", sections)
+
+				status.SetBackendTrafficPolicyCondition(policy,
+					egv1a1.PolicyConditionOverridden,
+					metav1.ConditionTrue,
+					egv1a1.PolicyReasonOverridden,
+					message,
+				)
+			}
+
+			// Add section to targeted gateways map
+			if s == nil {
+				targetedGateways[*gatewayKey] = sets.New[string]()
+			}
+			targetedGateways[*gatewayKey].Insert(AllSections)
+
+			translateBackendTrafficPolicy(policy, xdsIR)
+
+			// Set Accepted=True
+			status.SetBackendTrafficPolicyCondition(policy,
+				gwv1a2.PolicyConditionAccepted,
+				metav1.ConditionTrue,
+				gwv1a2.PolicyReasonAccepted,
+				"BackendTrafficPolicy has been accepted.",
+			)
+		}
+	}
+
+	return res
+}
+
+func resolveBTPolicyTargetRef(policy *egv1a1.BackendTrafficPolicy, gateways map[types.NamespacedName]*GatewayContext, routes map[types.NamespacedName]RouteContext) *types.NamespacedName {
+	targetNs := policy.Spec.TargetRef.Namespace
+	// If empty, default to namespace of policy
+	if targetNs == nil {
+		targetNs = ptr.To(gwv1b1.Namespace(policy.Namespace))
+	}
+
+	// Ensure Policy and target are in the same namespace
```

Comment:
curious if this can be moved to CEL Validation, not a blocker, we can raise a GH issue to track this